main.go

```diff
@@ -39,6 +40,14 @@ func main() {
 		config.Clustering.RaftBootstrap = c.Bool("raft-bootstrap-seed")
 		config.Clustering.RaftBootstrapPeers = c.StringSlice("raft-bootstrap-peers")
 
+		// NATS server specified on the command line takes precedence
+		if natsServer := c.String("nats-server"); natsServer != "" {
+			if config.NATS.Servers != nil {
+				fmt.Fprintf(app.Writer, "WARNING: overwriting NATS server(s) specified in config file %#v with command line value %s\n", config.NATS.Servers, natsServer)
+			}
+			config.NATS.Servers = []string{natsServer}
```

Comment:
We should allow passing in a list of servers from the CLI. For example: 

```
$ liftbridge --nats-server nats://localhost:4222,nats://localhost:4223,nats://localhost:4224
```

Reply:
Ah, of course... slaps forehead.

Can we rename the flag to *--nats-servers* to more accurately reflect this and bring it into line with the config file? 

If so, I'll update the doco, etc to match as part of this PR.

Reply:
Yes, we should rename the flag to `--nats-servers`. Same goes for the config file setting.

Reply:
Actually, looks like the config file option is already `servers`, so no changes there.

Reply:
Ok, this is done. As per the comments I had to workaround some issues in urfave/cli :-(